ci/generate-workflows.py

```diff
@@ -758,6 +758,9 @@ def update_homebrew_tap(self):
         return steps
 
     def global_env(self):
+        self.env["CARGO_INCREMENTAL"] = "0"
+        self.env["SCCACHE_GHA_ENABLED"] = "true"
+        self.env["RUSTC_WRAPPER"] = "sccache"
```

Comment:
There is probably a better way of doing this but the whole script is a big hack so...